### PR TITLE
add changelog

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## [Unreleased] - YYYY-MM-DD
+
+Optional release notice.
+
+- [Verb] Change description ([#<PR-number>](https://github.com/quiltdata/iac/pull/<PR-number>))
+-->
+
+## [1.0.0] - 2024-12-09
+
+- [Added] Add changelog ([#74](https://github.com/quiltdata/iac/pull/74))
```
Comment (sir-sigurd) [inline CHANGELOG.md:13]: I'm not sure yet it makes sense to have verbs

**Comment by sir-sigurd** [comment]
I'm not sure it's right to have a single changelog because we provide 2 terraform modules for users:

https://github.com/quiltdata/iac/blob/0ca3e1319cc89557ea31b3553012562d0e9a0b81/examples/main.tf#L32
https://github.com/quiltdata/iac/blob/0ca3e1319cc89557ea31b3553012562d0e9a0b81/examples/main.tf#L73

but in case of having separate changelogs for these modules I'm not sure how to organize releases

File: CHANGELOG.md (hunk 1 above)
Comment (nl0) [reply CHANGELOG.md:13]: sure we can live w/o them, just mark breaking changes somehow

**Comment by nl0** [comment]
> I'm not sure it's right to have a single changelog because we provide 2 terraform modules for users:
> 
> https://github.com/quiltdata/iac/blob/0ca3e1319cc89557ea31b3553012562d0e9a0b81/examples/main.tf#L32
> 
> https://github.com/quiltdata/iac/blob/0ca3e1319cc89557ea31b3553012562d0e9a0b81/examples/main.tf#L73
> 
> but in case of having separate changelogs for these modules I'm not sure how to organize releases

well, having two separate changelogs with two independent release lines sounds okay to me.
otoh, having a single release for all the (sub)modules doesn't sound bad either =)
it's your call (whatever makes more sense to you), since you're the one maintaining this.
i think it boils down to whether or not we intend to release those (sub) modules separately.

**Comment by sir-sigurd** [comment]
> it's your call (whatever makes more sense to you), since you're the one maintaining this.
> i think it boils down to whether or not we intend to release those (sub) modules separately.

well, I don't know 😄 
in case of separate releases I guess we'd have to add a component prefix for git tags
so having a single release seems simpler
also I think instead of suggesting using `cnames` module we could just add an option for the 'main' `quilt` module

**Review by nl0 — APPROVED** [review]

**Pull request merged**